examples/ivr/playRecording.js

```diff
@@ -0,0 +1,24 @@
+// play ivr menu
+const ivrMenu = (conn) => {
+  try {
+    conn.execute('playback', '/usr/share/freeswitch/sounds/en/us/callie/misc/48000/transfer1.wav');
```

Comment:
Great point. I added the files locally in the project directory. This way they will be available right away when someone clones. I also added a comment on where to find more recordings if anyone needs them. 

Does that work?